### PR TITLE
Add dynamic routes example in isr.exclude for @astrojs/vercel integration

### PR DESCRIPTION
**Pull request by tk04**
#### Description (required)

Added an example that shows how you would exclude dynamic routes from ISR.
#### Related issues & labels (optional)

- Closes # 
- Suggested label:

### CODE REVIEW

File: src/content/docs/en/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -274,7 +274,7 @@ export default defineConfig({
 
 ##### Excluding paths from caching
 
-To implement Vercel's [Draft mode](https://vercel.com/docs/build-output-api/v3/features#draft-mode), or [On-Demand Incremental Static Regeneration (ISR)](https://vercel.com/docs/build-output-api/v3/features#on-demand-incremental-static-regeneration-isr), you can create a bypass token and provide it to the `isr` config along with any paths to exclude from caching: 
+To implement Vercel's [Draft mode](https://vercel.com/docs/build-output-api/v3/features#draft-mode), or [On-Demand Incremental Static Regeneration (ISR)](https://vercel.com/docs/build-output-api/v3/features#on-demand-incremental-static-regeneration-isr), you can create a bypass token and provide it to the `isr` config along with any static or dynamic routes to exclude from caching:
```
Comment (lilnasy) [inline src/content/docs/en/guides/integrations-guide/vercel.mdx:277]: "static" and "dynamic" are overloaded terms. In this case we can probably avoid them altogether, and let the example below communicate the fact.
```suggestion
To implement Vercel's [Draft mode](https://vercel.com/docs/build-output-api/v3/features#draft-mode), or [On-Demand Incremental Static Regeneration (ISR)](https://vercel.com/docs/build-output-api/v3/features#on-demand-incremental-static-regeneration-isr), you can create a bypass token and provide it to the `isr` config along with any routes to exclude from caching:
```

**Review by lilnasy — APPROVED** [review]
Looks great to me content-wise. Just a small suggestion regarding communication.

**Comment by sarah11918** [comment]
Thank you for this contribution @tk04 ! I am going to ask @lilnasy to review this one when he's available!

File: src/content/docs/en/guides/integrations-guide/vercel.mdx (hunk 1 above)
Comment (tk04) [reply src/content/docs/en/guides/integrations-guide/vercel.mdx]: Good point 👍

**Pull request merged**